media/error-message.js

```diff
@@ -17,3 +17,13 @@ nonExistentFileButton.addEventListener('click', function() {
 invalidFileButton.addEventListener('click', function() {
   video.src = 'https://storage.googleapis.com/media-error/no_streams.webm';
 });
+
+chooseFileButton.addEventListener('change', function() {
+  const file = chooseFileButton.files[0];
+  video.src = URL.createObjectURL(file);
+});
+
+video.addEventListener('canplay', function() {
+  ChromeSamples.log('Video can play!');
```

Comment:
Nit: I suggest something more like "'canplay' event received" instead, because:
1) it might not be a video
2) "Video can play!" might overstate the ability of the pipeline to actually decode and render the media.

Otherwise LGTM.